ci-operator/step-registry/baremetal/lab/ipi/virtual-media/dualstack/baremetal-lab-ipi-virtual-media-dualstack-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: baremetal-lab-ipi-virtual-media-dualstack
+  steps:
+    pre:
+      - chain: baremetal-lab-ipi-conf-virtual-media
+      - ref: baremetal-lab-ipi-conf-network-vips
```

Comment:
this should be added to all the BM IPI workflows. Also, we may skip creating a dedicated workflow for the 'combination' (virtual-media, dual-stack)

Reply:
I rechecked and found that I added it to:
baremetal-lab-ipi-workflow.yaml
baremetal-lab-ipi-external-lb-workflow.yaml
baremetal-lab-ipi-virtual-media-workflow.yaml
baremetal-lab-ipi-virtual-media-dualstack-workflow.yaml